Extensions/GoogleOptimize/Src/README.md

```diff
@@ -0,0 +1,41 @@
+# Google Optimize
+Optimize allows you to test variants of web pages and see how they perform against an objective that you specify. Optimize monitors the results of your experiment and tells you which variant is the leader. To get started:
```

Comment:
I assume the Readme contents are reviewed by PMs. If not, please get their input.